apps/agiflow-sdk-samples/README.md

```diff
@@ -0,0 +1,26 @@
+# Agiflow-sdk samples
+
+Run script to test agiflow-sdks.
+
+
+## Getting started
+
+Install mono-repo dependencies
+
+``` sh
+pnpm install
+```
+
+Then activate virtual environment
+
+``` sh
+poetry shell
+```
+
+## Test scripts
+
+### OpenAI Ollama
+
+``` sh
+python run app/openai_ollama.py
```

Comment:
```suggestion
python app/openai_ollama.py
```

Reply:
Thanks @codefromthecrypt  for the details, make my updates easier :). 

Completion and prompt span events are now supported. If you don't have other concern, I'll merge this as soon as our backend support changes in span events.

<img width="905" alt="Screenshot 2024-08-05 at 10 33 32 pm" src="https://github.com/user-attachments/assets/4f37c623-d8d3-4e68-b707-3f5aefdaa28e">

Reply:
Thanks, this has been updated!